architecture/jax/minimal.py

```diff
@@ -0,0 +1,85 @@
+# ************************************************************************
+# FAUST Architecture File
+# Copyright (C) 2022 GRAME, Centre National de Creation Musicale
+# ---------------------------------------------------------------------
+
+# This is sample code. This file is provided as an example of minimal
+# FAUST architecture file. Redistribution and use in source and binary
+# forms, with or without modification, in part or in full are permitted.
+# In particular you can create a derived work of this FAUST architecture
+# and distribute that work under terms of your choice.
+
+# This sample code is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# ************************************************************************
+
```

Comment:
Could this file contain `load_soundfile/add_soundfile/add_nentry/add_button/add_slider` functions?